src/AddTodoForm.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react'
 
-function AddTodoForm() {
+function AddTodoForm(props) {
+      const handleAddTodo = (event) =>{ 
+        event.preventDefault();
+        const todoTitle = event.target.title.value;
+        console.log(todoTitle);
+        props.onAddTodo(todoTitle + " is the result")
+      }
```

Comment:
Requires more work:
When the form is submitted, this callback function should:
> Finally, reset the form so the text input value is cleared

In this case, since you're not controlling the input's value using React state, you'll want to reset the form **on submit**. For reference, please visit this link on how to [reset a form](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset) programmatically using a [document query selector](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset#examples).

<details>
<summary>Click here for answer if you're stuck</summary>

```js
const handleAddTodo = (event) =>{ 
  event.preventDefault();
  const todoTitle = event.target.title.value;
  props.onAddTodo(todoTitle + " is the result");

  // be careful about using "form" as a query selector!
  // if you have multiple forms in the DOM, then this
  // require more specificity because the
  // querySelector finds the first "form" element in 
  // the DOM starting from the top! To mitigate this,
  // you can add an "id" attribute to the form element
  // below and use that id as a selector instead.
  document.querySelector("form").reset();
}
```
</details>